modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/custom/form-report/index.scss

```diff
@@ -40,31 +59,40 @@
 	.report-cards-shortcut {
 		display: none;
 	}
+}
 
-	.custom-chart-size {
-		&.pie-chart {
-			display: block;
+.col-md-12 {
+	.lfr-de__form-report {
+		.report-cards-area {
+			width: 100%;
 		}
-	}
-}
 
-.col-md-4 {
-	.custom-chart-size {
-		&.pie-chart {
-			display: block;
-			margin-left: -83px;
+		.report-cards-shortcut {
+			display: none;
+		}
+
+		@media (min-width: 1280px) {
+			.report-cards-area {
+				width: auto;
+
+				.card-item {
+					width: 808px;
+				}
+			}
+
+			.report-cards-shortcut {
+				display: block;
+			}
```

Comment:
Is okay to use the media query like this inside the `col-md-12` class? Usually what I see is:
```

@media (min-width: 992px) {
  .col-md-12 {
    width: 100%;
  }
}
```

Reply:
Maybe instead of using the media query inside the class we should use it at the end of the file with the `.col-md-12` inside the media query? I will search in our frontend guidelines about this

Reply:
Hi @rebsilva. Good observation!
It works since we use sass. In pure css it wouldn't work. https://css-tricks.com/media-queries-sass-3-2-and-codekit/
I've seen other products doing the same, so I think it's not a problem.

Reply:
You are right! Is really good to know this, thanks a lot! :purple_heart: